pybamm/expression_tree/symbol.py

```diff
@@ -27,6 +27,8 @@ def domain_size(domain):
         "negative electrode": 11,
         "separator": 13,
         "positive electrode": 17,
+        "working electrode": 19,
+        "working particle": 23,
```

Comment:
I think this has to do because the separator and current collector were defined in `fixed_domain_sizes` but the new domains weren't, so when they interact it breaks down...

Also, random question related to that. When we get the size of a variable defined in a domain in `fixed_domain_sizes`, why do we take the sum and not the product? I thought the idea of assigning prime numbers to the domains was to get unique signatures for each combination.